### ISSUE TITLE
Can´t open the app [Bug report]: 

### ISSUE THREAD

**Issue by tbrites** [iotaledger/firefly#7343]
### Platform

Desktop

### Version

1.7.3

### Description

I can´t open the app firefly v1.7.3

### Expected behaviour

APP working 

### Actual behaviour

Not working the app

### Can the issue reliably be reproduced?

Yes

### Steps to reproduce the issue

Open the app
Second, give me this issue:
App Version: 1.7.3

platform: macOS
platformVersion: 10.12.6
platformArchitecture: x64
cpuCount: 4
totalMem: 16384.0 MB
freeMem: 6154.1 MB

Error Type: [Preload Context] Error

Error: node-loader:
Error: dlopen(/Applications/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node, 1): Symbol not found: __ZNSt3__115basic_stringbufIcNS_11char_traitsIcEENS_9allocatorIcEEE7seekoffExNS_8ios_base7seekdirEj
  Referenced from: /Applications/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node
  Expected in: /usr/lib/libc++.1.dylib
 in /Applications/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node

### Operating System

Mac

### Diagnostics

```shell
App Version: 1.7.3

platform: macOS
platformVersion: 10.12.6
platformArchitecture: x64
cpuCount: 4
totalMem: 16384.0 MB
freeMem: 6154.1 MB

Error Type: [Preload Context] Error

Error: node-loader:
Error: dlopen(/Applications/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node, 1): Symbol not found: __ZNSt3__115basic_stringbufIcNS_11char_traitsIcEENS_9allocatorIcEEE7seekoffExNS_8ios_base7seekdirEj
  Referenced from: /Applications/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node
  Expected in: /usr/lib/libc++.1.dylib
 in /Applications/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node
```

### Developer Profile

No

### Profile Type

Software (Stronghold)

### Staking

Actively staking

### Crash Status

White screen

### Error Logs

```shell
App Version: 1.7.3

platform: macOS
platformVersion: 10.12.6
platformArchitecture: x64
cpuCount: 4
totalMem: 16384.0 MB
freeMem: 6154.1 MB

Error Type: [Preload Context] Error

Error: node-loader:
Error: dlopen(/Applications/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node, 1): Symbol not found: __ZNSt3__115basic_stringbufIcNS_11char_traitsIcEENS_9allocatorIcEEE7seekoffExNS_8ios_base7seekdirEj
  Referenced from: /Applications/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node
  Expected in: /usr/lib/libc++.1.dylib
 in /Applications/Firefly.app/Contents/Resources/app.asar.unpacked/public/build/index.node
```

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by begonaalvarezd** [comment]
Closed by https://github.com/iotaledger/firefly/issues/7324

**Comment by tbrites** [comment]
Hello!
I updated the version Catalina but I still have the same issue.
When is the new version out?
Best regards.
TB

**Comment by begonaalvarezd** [comment]
Hi, we just released the new version 1.7.4 which contains the fix

### Firefly IOTA 1.7.4 announcement message

We just released Firefly IOTA 1.7.4. Among the changes, it contains the fixes for macOS crash at startup and the slow stronghold export. 

:warning: Important: macOS 10.13 will not be supported anymore in Firefly IOTA, so if you are running macOS 10.13 we highly recommend upgrading your macOS version to a newer one. 

:pencil: Changelog:

- fix slow stronghold export
- fix macOS support (official drop support macOS <= 10.13)
- add official asmb staking event 6 results

:arrow_down:  Download:
https://firefly.iota.org/

:bug:  Bug Reporting:
https://github.com/iotaledger/firefly/issues/new?assignees=&labels=bug+report&template=bug_report.yml&title=%5BBug+report%5D%3A
